### DISCUSSION TITLE
plot MarkerFeatures Heatmaps/ComplexHeatmaps of each sample in an object?

### DISCUSSION THREAD

**Discussion by matteozoia4** [GreenleafLab/ArchR#1627] [Questions / Documentation]
My object contains 3 samples.

Which approach would be the best one to getMarkerFeatures (per clusters) for all the samples (all in once)?

Allowing then to choose a sample and plot relatives clusters heatmap/complexheatmap?

Many thanks for the help!

MZ

**Comment by rcorces** [comment]
See manual
https://www.archrproject.com/bookdown/identifying-marker-genes.html
